woocommerce/woocommerce-ios
# Update illustrations for orders tab and products tab empty search results

Part of #3786 

## Why

We're ready to update the illustrations in the app to the latest design from core! This PR focused on the orders tab, and the change of empty search results also applies to products tab empty search results.

## Changes

Updated illustrations in:
- Orders tab:
  - processing empty state
  - all orders empty state
  - empty search results
  - empty filtered results
- Products tab:
  - empty search results

## Testing

You can hard code to show the empty states for orders tab by updating the siteID to a different value (e.g. 123):

https://github.com/woocommerce/woocommerce-ios/blob/876b67c9a4dac7f5c46de995995d213fd4e17fde/WooCommerce/Classes/ViewRelated/Orders/OrderListViewModel.swift#L150

- Launch the app and log in if needed
- Go to the Orders tab where both tabs have no data --> the empty state image should look as expected in both dark and light mode
- Tap the search icon
- Tap into an order status row that has 0 orders --> the empty state image should look as expected in both dark and light mode
- Navigate back to the search screen, and enter some query that renders no results --> the empty state image should look as expected in both dark and light mode
- Dismiss the orders search
- Go to the Products tab
- Tap the search icon
- Enter some query that renders no results --> the empty state image should look as expected in both dark and light mode

## Example screenshots

### Dark mode

\ | Orders - Processing | Orders - All Orders | Orders search | Filtered orders | Products search
-- | -- | -- | -- | -- | --
before | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 33 32](https://user-images.githubusercontent.com/1945542/110426454-18fa8400-80e1-11eb-905b-711cbcdcb0cb.png) | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 33 36](https://user-images.githubusercontent.com/1945542/110426458-1bf57480-80e1-11eb-8a5e-cbbcd308e5a5.png) | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 34 06](https://user-images.githubusercontent.com/1945542/110426461-1d26a180-80e1-11eb-95b7-2f2415310612.png) | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 34 41](https://user-images.githubusercontent.com/1945542/110426464-1d26a180-80e1-11eb-8748-f99652c25fec.png) | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 48 30](https://user-images.githubusercontent.com/1945542/110426466-1dbf3800-80e1-11eb-910e-d25cce545b88.png)
after | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 53 26](https://user-images.githubusercontent.com/1945542/110426503-30397180-80e1-11eb-9700-ada6e76b19fd.png) | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 53 29](https://user-images.githubusercontent.com/1945542/110426511-329bcb80-80e1-11eb-9d09-3d47184cdc0b.png) | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 53 36](https://user-images.githubusercontent.com/1945542/110426514-33ccf880-80e1-11eb-9eb6-008b3671a955.png) | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 53 41](https://user-images.githubusercontent.com/1945542/110426516-34658f00-80e1-11eb-86c2-b06c6c4a1435.png) | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 53 48](https://user-images.githubusercontent.com/1945542/110426518-34fe2580-80e1-11eb-81a0-4015ff7262a9.png)


### Light mode

\ | Orders - Processing | Orders - All Orders | Orders search | Filtered orders | Products search
-- | -- | -- | -- | -- | --
before | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 34 57](https://user-images.githubusercontent.com/1945542/110426599-5f4fe300-80e1-11eb-9446-da22bac47adf.png) | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 34 59](https://user-images.githubusercontent.com/1945542/110426606-61b23d00-80e1-11eb-9061-6bc9e8a32dff.png) | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 35 05](https://user-images.githubusercontent.com/1945542/110426613-62e36a00-80e1-11eb-94aa-958e2e820ec9.png) | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 35 09](https://user-images.githubusercontent.com/1945542/110426616-637c0080-80e1-11eb-97a4-312a46920bcc.png) | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 48 20](https://user-images.githubusercontent.com/1945542/110426617-637c0080-80e1-11eb-8a53-2c2dab99379f.png)
after | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 53 59](https://user-images.githubusercontent.com/1945542/110426648-6ecf2c00-80e1-11eb-81ff-8f3b4fe57199.png) | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 54 02](https://user-images.githubusercontent.com/1945542/110426653-71318600-80e1-11eb-98cf-0071482e9b2f.png) | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 54 08](https://user-images.githubusercontent.com/1945542/110426661-742c7680-80e1-11eb-8d8d-feffdc6a2bcd.png) | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 54 30](https://user-images.githubusercontent.com/1945542/110426666-74c50d00-80e1-11eb-82e8-c0508b02f58c.png) | ![Simulator Screen Shot - iPhone 11 - 2021-03-09 at 13 54 35](https://user-images.githubusercontent.com/1945542/110426668-75f63a00-80e1-11eb-9dde-ef30ddfb4448.png)
















Update release notes:
